basics/installation/nginx.md

```diff
@@ -134,54 +92,53 @@ server {
     rewrite ^/c/([0-9]+)(-[.*_a-zA-Z0-9-]*)(-[0-9]+)?/.+.jpg$ /img/c/$1$2$3.jpg last;
     rewrite ^/c/([a-zA-Z_-]+)(-[0-9]+)?/.+.jpg$ /img/c/$1$2.jpg last;
 
-    # AlphaImageLoader for IE and fancybox
-    rewrite ^images_ie/?([^/]+)\.(jpe?g|png|gif)$ js/jquery/plugins/fancybox/images/$1.$2 last;
+    # AlphaImageLoader for IE and FancyBox.
+    rewrite ^images_ie/?([^/]+)\.(gif|jpe?g|png)$ js/jquery/plugins/fancybox/images/$1.$2 last;
 
-    # Web service API
+    # Web service API.
     rewrite ^/api/?(.*)$ /webservice/dispatcher.php?url=$1 last;
 
-    # Installation sandbox
+    # Installation sandbox.
     rewrite ^(/install(?:-dev)?/sandbox)/(.*) /$1/test.php last;
-    
-    # without this line nginx seo url not working.
-    try_files $uri $uri/ /index.php?$args;
-     
-    # [REQUIRED EDIT] Change this block to your admin folder
+
+    location / {
+        try_files $uri $uri/ /index.php$is_args$args;
+    }
+
+    # [EDIT] Replace 'admin-dev' in this block with the name of your admin directory.
     location /admin-dev/ {
         if (!-e $request_filename) {
             rewrite ^/.*$ /admin-dev/index.php last;
         }
     }
 
-
-    # File security
-    # .htaccess .DS_Store .htpasswd etc
+    # .htaccess, .DS_Store, .htpasswd, etc.
     location ~ /\. {
         deny all;
     }
-    # Source code directories
-    location ~ ^/(app|bin|cache|classes|config|controllers|docs|localization|override|src|tests|tests-legacy|tools|translations|travis-scripts|vendor|var)/ {
+
+    # Source code directories.
+    location ~ ^/(app|bin|cache|classes|config|controllers|docs|localization|override|src|tests|tools|translations|travis-scripts|var|vendor)/ {
```

Comment:
```suggestion
    location ~ ^/(app|bin|cache|classes|config|controllers|docs|localization|override|src|tests|translations|var|vendor)/ {
```

There are no tools or travis-scripts in recent PrestaShop installation (or in releases)